homeassistant/components/valve/__init__.py

```diff
@@ -183,9 +183,9 @@ def state(self) -> str | None:
         if reports_position is True:
             if (current_valve_position := self.current_valve_position) is None:
                 return None
-            closed = current_valve_position == 0
+            closed: bool | None = current_valve_position == 0
```

Comment:
This can't be `None`. Use another name for the variable if there's a typing conflict.

Reply:
Good point. Fixed